app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChargesController < ApplicationController
+	def new
```

Comment:
You've got an indentation error here; everything inside the class should start at 2 spaces indented.